library/item/item_group/src/main/java/org/quiltmc/qsl/itemgroup/impl/CreativeGuiExtensions.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.itemgroup.impl;
+
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public interface CreativeGuiExtensions {
+	void qsl$nextPage();
```

Comment:
```suggestion
	void quilt$nextPage();
```

Sorry for not mentioning this earlier, I've missed it, but CONTRIBUTING.md says we should use the quilt prefix instead of qsl.